docker-compose.override.yml

```diff
@@ -0,0 +1,10 @@
+version: "3.9"
```

Comment:
What does this file do? How to execute it?

Reply:
> By default, Compose reads two files, a docker-compose.yml and an optional docker-compose.override.yml file. By convention, the docker-compose.yml contains your base configuration. 

From [docker-compose documentation](https://docs.docker.com/compose/extends/). So you could e.g. run `docker-compose up` for development and `docker-compose -f docker-compose.yml up` to simulate production.

Reply:
By the way: Who is going to merge? The reviewer or the author?